src/assets/config/version.txt

```diff
@@ -0,0 +1,2 @@
+This file will be generated by the pipeline in order to store the current version of this build
```

Comment:
The GitHub Pipeline `.github/workflows/build-and-release-image.yaml` will replace this file during build time with...

This file will be served to allow checking for the exact source code version of a deployed edc-ui.